test.js

```diff
@@ -194,6 +210,6 @@ test('prototype support', t => {
 
 test('mem.clear() throws when called with a plain function', t => {
 	t.throws(() => {
-		mem.clear(() => {});
+		mem.clear(() => { });
```

Comment:
Can you disable Prettier for non-owned projects? I don't want to have to comment on this on every PR.

Reply:
Prettier is already off. I don't get how vscode is still doing this.

Reply:
Try disabling all plugins and then enable them one-by-one until it happens again.

Reply:
Found it in the built-in formatter options:

![JavaScript › Format: Insert Space After Opening And Before Closing Nonempty Braces - Defines space handling after opening and before closing non-empty braces. Requires using TypeScript 2.3.0 or newer in the workspace.](https://user-images.githubusercontent.com/29491356/90838966-74199300-e3aa-11ea-889e-f940120ee919.png)